data/remnant/remnant missions.txt

```diff
@@ -4479,3 +4479,140 @@ mission "Remnant: Expanded Horizons Astral job"
+				goto stormend
+			label numericsystem
+			`	She looks confused for a moment before responding. "It is called hexadecimal. It is a base 16 numeric system." She pauses to look at you. "Based on your reaction, I assume that base 10 is still the standard in human space?"`
+			choice
+				`	"Yes."`
+					goto  stormcontinue
+				`	"How does that work?"`
+			`	"Quite simply: We have added some extra characters so that when counting instead of a single digit position restarting at 0 every 10, they cycle every 16. So when we count we start at 1, but when we reach 9, the next number will be A. Then B, C, D, E and F. Then it is at that point that we add an extra digit and restart at 0 with 10. So in hexadecimal F would be equal to 15 in base 10. One of the advantages of using a numeric system with a higher base is that we can handle larger numbers with fewer digits. For example, with two digits we can communicate any integer up to what would be 255 in base 10. Or with three digits we can reach 4095. Saving an extra digit here or there is fairly meaningless in written formats, nor is it much of a difference in oral communication. However, when one communicates by gestures it is helpful to shorten the numbers where we can."`
+			label stormcontinue
+			`	"Now that you understand our numbers a bit better, you will be able to recognize that there is a lot of unexplained energy being transfered out there."`
+			choice
+				`	"Right. Is that what causes the red glow?"`
+					goto stormresume
+				`	"Thanks! I think that is all the details I can handle right now."`
+					goto stormend
+			
+			label tempunits
+			`	She looks at you curiously. "Corrected degrees Kelvin, of course. Why?"`
+			choice
+				`	"What are 'Corrected degrees Kelvin'?"`
+				`	"Just clarifying. I prefer to use degrees Centigrade."`
+					goto centigrade
+				`	"Just clarifying. I prefer to use degrees fahrenheit."`
+					goto fahrenheit
+			`	"Corrected degrees Kelvin are basically the standard Kelvin scale developped millenia ago, but with a slight correction factor to account for the adjustments determined by the conjunction of 2730. It is a rather miniscule change to the scale that is mostly only applicable to experiments or industrial applications that require micro-degree temperature precision, but can be significant at the larger scales." The display and its field of stars vanishes as she slings her backpack over her shoulder.`
+				goto stormend
+			label centigrade
+			`	"People still use that? We learned about it in our studies, mostly in case we came across references to it in reference books. I have seen it a few times, but it is rarely used among us." The display and its field of stars vanishes as she slings her backpack over her shoulder.`
```

Comment:
Is there any way to get rid of the double `reference` so close together? Would taking `reference` out of `reference books` or changing it to `research books` change its meaning too significantly? Your call.

Reply:
Yeah, that can be changed. I'll fix it. Thanks!

Reply:
Fixed. Thanks for pointing this out. That phrase has been replaced by "in case we came across references to it in our libraries." Which conveys the same meaning, but sounds better and doesn't have the duplicate wording.